bootstrap/kubeadm/config/manager/manager.yaml

```diff
@@ -16,31 +16,43 @@ spec:
         control-plane: controller-manager
     spec:
       containers:
-      - command:
-        - /manager
-        args:
-        - "--leader-elect"
-        - "--metrics-bind-addr=localhost:8080"
-        - "--feature-gates=MachinePool=${EXP_MACHINE_POOL:=false},KubeadmBootstrapFormatIgnition=${EXP_KUBEADM_BOOTSTRAP_FORMAT_IGNITION:=false}"
-        - "--bootstrap-token-ttl=${KUBEADM_BOOTSTRAP_TOKEN_TTL:=15m}"
-        image: controller:latest
-        name: manager
-        ports:
-        - containerPort: 9440
-          name: healthz
-          protocol: TCP
-        readinessProbe:
-          httpGet:
-            path: /readyz
-            port: healthz
-        livenessProbe:
-          httpGet:
-            path: /healthz
-            port: healthz
+        - command:
+            - /manager
+          args:
+            - "--leader-elect"
+            - "--metrics-bind-addr=localhost:8080"
+            - "--feature-gates=MachinePool=${EXP_MACHINE_POOL:=false},KubeadmBootstrapFormatIgnition=${EXP_KUBEADM_BOOTSTRAP_FORMAT_IGNITION:=false}"
+            - "--bootstrap-token-ttl=${KUBEADM_BOOTSTRAP_TOKEN_TTL:=15m}"
+          image: controller:latest
+          name: manager
+          ports:
+            - containerPort: 9440
+              name: healthz
+              protocol: TCP
+          readinessProbe:
+            httpGet:
+              path: /readyz
+              port: healthz
+          livenessProbe:
+            httpGet:
+              path: /healthz
+              port: healthz
+            securityContext:
```

Comment:
Q: (for my understanding) do we need `allowPrivilegeEscalation: false` and `privileged: false` (which AFAIK are the the default for those fields)

Reply:
IIRC the securityContexts on pod and container level are not the same (i.e. you can't set the same fields in both)

Reply:
What @sbueringer says is correct. `allowPrivilegeEscalation` would require a capability to escalate via sudo, but the point stands.  Privileged container would be effective root, and you wouldn't be able these limitations to privileged container anyways.

Whole point of the PR is to make security configuration explicit, and not rely on defaults or fallbacks. As mentioned in PR description, none of these (besides seccompPolicy, but see earlier comments how it is in-fact quite no-change as well) actually change things, they make it explicit and enforce them.